metacat-connector-polaris/src/functionalTest/java/com/netflix/metacat/connector/polaris/PolarisStoreConnectorFunctionalTest.java

```diff
@@ -18,4 +23,65 @@
 @AutoConfigureDataJpa
 public class PolarisStoreConnectorFunctionalTest extends PolarisStoreConnectorTest {
 
+    /**
+     * Test getTableEntities.
+     */
+    @Test
+    public void testGetTableEntities() {
+        // Create the db
+        final String dbName = generateDatabaseName();
+        createDB(dbName);
+
+        try {
+            // pause execution for 10000 milliseconds (10 seconds)
+            Thread.sleep(10000);
+        } catch (InterruptedException e) {
+            System.out.println("Sleep was interrupted");
+        }
+
+        // Test when db is empty
+        List<PolarisTableEntity> entities = getPolarisConnector().getTableEntities(dbName, "", 1);
+        Assert.assertEquals(0, entities.size());
+
+
+        // Add some tables
+        final String tblNameA = "A_" + generateTableName();
+        final String tblNameB = "B_" + generateTableName();
+        final String tblNameC = "C_" + generateTableName();
+        createTable(dbName, tblNameA);
+        createTable(dbName, tblNameB);
+        createTable(dbName, tblNameC);
+
+        try {
+            // pause execution for 10000 milliseconds (10 seconds)
+            Thread.sleep(10000);
```

Comment:
Is the purpose of sleep to wait for the create to complete, the create call should be synchronous and return the created object after it is done

Reply:
Since we are now only reading the snapshot that is follower_reader_timestamp (4.8s) ago, we need to sleep for sometime so the following list operation can read the 3 tables created.